### ISSUE TITLE
TFT driver wave35

### ISSUE THREAD

**Issue by fachos** [Re4son/Re4son-Pi-TFT-Setup#15]
Hi... i managed to install wave35(A) driver and the screen come out... the problem is... cant touch or use the screen... im using kali-pi and follow all your step... please guide me... thanks

**Comment by d3c3ptic0n** [comment]
Yes same problem here...Display works but touch screen doesnt....Any fix for this?
